ckanext/bcgov_schema/bcdc_dataset.json

```diff
@@ -26,6 +26,54 @@
       "required": true,
       "help_text": "Custodial business responsible for the record and resources."
     },
+    {
+      "field_name": "publish_state",
+      "preset": "state_machine",
+      "label": "State",
+      "startState": {"state": "DRAFT", "by": ["editor", "admin", "sysadmin"]},
+      "required": true,
+      "help_text": "For new records select Draft. To change: edit, select new state and save.",
```

Comment:
New records should be set to Draft. To change state: edit, select new state and save.